lib/os_mon/doc/src/disksup.xml

```diff
@@ -111,6 +111,46 @@
           available.</p>
       </desc>
     </func>
+    <func>
+      <name since="">get_disk_info() -> [DiskData]</name>
+      <fsummary>Immediately get information for the disks in the system</fsummary>
+      <type>
+        <v>DiskData = {Id, TotalKiB, AvailableKiB, Capacity}</v>
+        <v>&nbsp;Id = string()</v>
+        <v>&nbsp;TotalKiB = int()</v>
+        <v>&nbsp;AvailableKiB = int()</v>
+        <v>&nbsp;Capacity = int()</v>
+      </type>
+      <desc>
+        <p>Immediately fetches total space, available space and capacity for local disks.
+          <c>Id</c> is a string that identifies the disk or partition.
+          <c>TotalKiB</c> is the total size of the disk or partition in kbytes.
+          <c>AvailableKiB</c> is the disk space used in kbytes.
+          <c>Capacity</c> is the percentage of disk space used.</p>
+        <p>Returns <c>[{"none",0,0,0}]</c> if <c>disksup</c> is not
```

Comment:
Would it be better if we return an empty list or an error in this case instead?

Reply:
🤷 that's how `disksup` works now. My goal is to not change the existing API.